### ISSUE TITLE
Feature Request - Income minus expenses

### ISSUE THREAD

**Issue by StrayaJake** [yet-another-tool/obsidian-findoc#9]
A nice setting where we could compare an Income total sum and take expenses total sum, or have it so we could have a prime number say income, that takes multiple total sums say house mortgage and expenses

IE
```
Income: $5000
Expesnse: $3000
Chart/Report would display $2000
```

**Comment by studiowebux** [comment]
Hello, 
Just to let you know I've started few days ago the v0.6.0 refactoring to include this feature

**Comment by studiowebux** [comment]
Hello, 

I merged v0.6.0

Due to breaking changes, you have to overwrite the data.json file (Instruction here: https://studiowebux.github.io/obsidian-plugins-docs/docs/findoc/defaultConfigurations/)

Please feel free to reopen the ticket if there is anything
